docs/source/conf.py

```diff
@@ -441,13 +452,61 @@ def check_timeout_and_api_kwargs_presence(obj: object) -> int:
     )
 
 
+def create_return_admonitions() -> dict[str, str]:
+    """Creates 'Returned in' admonitions for classes that are returned in Bot's methods."""
+
+    # First, generate a mapping of class names to the methods which return it,
+    # i.e. {Message: [send_message, ...]}
+    methods_for_class_name = defaultdict(list)
+
+    for method in [
+        m[0]
+        for m in inspect.getmembers(telegram.Bot, inspect.isfunction)  # not .ismethod
+        if not m[0].startswith("_") and m[0].islower()  # islower() to avoid camelCase methods
+    ]:
+        sig = inspect.signature(getattr(telegram.Bot, method))
+        classes = get_args(sig.return_annotation)
+
```

Comment:
```suggestion
        sig = inspect.signature(getattr(telegram.Bot, method))
        ret_anot = sig.return_annotation
        classes = get_args(ret_anot)

        if not classes and isinstance(ret_anot, type) and "telegram" in str(ret_anot):
            methods_for_class_name[str(ret_anot)].append(method)
            continue
```

Found something I didn't expect with get_args. If you do `get_args(telegram.Message)`, it returns `()`. So we can do this workaround to fix it.

Reply:
Done